src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -1147,13 +1147,23 @@ static BoundCollectionExpressionSpreadElement bindSpreadElement(
                         ErrorCode.ERR_CollectionArgumentsNotSupportedForType,
                         _node.WithElement.Syntax.GetFirstToken().GetLocation(),
                         _targetType);
+                    return null;
```

Comment:
>return null; [](http://example.com/codeflow?start=20&length=12)

We should have a test verifying what happens with `WithElement` in IOperation tree when we hit this code path.

Reply:
> 511ca836530

Could you please provide the name of the test?